static_frame/core/store.py

```diff
@@ -216,11 +232,20 @@ def __init__(self,
                 if (config.label_encoder != self._default.label_encoder or
                         config.label_decoder != self._default.label_decoder):
                     raise ErrorInitStoreConfig(f'config {label} has encoder/decoder inconsistent with default; align values and/or pass a default StoreConfig.')
+                if (config.read_max_workers != self._default.read_max_workers or
+                        config.read_chunksize != self._default.read_chunksize):
+                    raise ErrorInitStoreConfig(f'config {label} has read_max_workers/chunksize inconsistent with default; align values and/or pass a default StoreConfig.')
+                if (config.write_max_workers != self._default.write_max_workers or
+                        config.write_chunksize != self._default.write_chunksize):
+                    raise ErrorInitStoreConfig(f'config {label} has write_max_workers/chunksize inconsistent with default; align values and/or pass a default StoreConfig.')
                 self._map[label] = config
 
     def __getitem__(self, key: tp.Optional[tp.Hashable]) -> StoreConfig:
         return self._map.get(key, self._default)
 
+    def is_empty(self) -> bool:
+        return len(self._map) == 0
+
```

Comment:
I do not think we need this, as we would generally just use the default config if no per-frame configs are provided.